src/clinical/service-worker-thread/runner.ts

```diff
@@ -0,0 +1,17 @@
+import { StaticPool } from 'node-worker-threads-pool';
+import { WorkerTasks } from './tasks';
+
+const poolSize = 1;
```

Comment:
why 1 ?

Reply:
Yea I was just thinking about this now. Maybe this should be configurable??

Reply:
No real reason for one. But I imagine under loads might need more then that

Reply:
it can be max the number of cores given, there is a config for lectern parallel, maybe you can repurpose that config

Reply:
good point, we can just use that :+1: